docs/user/convert/json.rst

```diff
@@ -9,12 +9,10 @@ definition.
 Configuration
 -------------
 
-The JSON converter supports parsing files in single-line mode or in multi-line mode. In single-line mode, each line
-of an input file should be a valid JSON document; in multi-line mode, the entire input file should be a single valid
-JSON document. In order to support JSON path expressions, each JSON document is fully parsed into memory.
-For large documents, this may take considerable time and memory. Thus, it is usually better to use single-line mode
-when possible. Line mode may be specified by ``options.line-mode = "single"`` or ``options.line-mode = "multi"``
-in your converter definition. If nothing is specified, single-line mode is used.
```

Comment:
Ok, is there still a single-line and multi-line option here?

Reply:
not for json, since we are reading it as an input stream we can just parse out the elements as they come, regardless of line breaks